src/LightPanels/LightPanels.js

```diff
@@ -531,7 +531,7 @@ module.exports = kind(
 		commonInfo = {addBefore: insertBefore};
 
 		// remove existing panels
-		for (idx = start; idx < end; idx++) {
+		for (idx = end - 1; idx >= start; idx--) {
```

Comment:
If count is 0 (or < 0) then idx is going to start out < start and go down from there. Do we want to guard against this?